netbox/core/apps.py

```diff
@@ -22,3 +23,6 @@ def ready(self):
 
         # Register models
         register_models(*self.get_models())
+
+        # Clear Swagger API cache on startup
+        cache.delete("*api_schema_*")
```

Comment:
This does not appear to work:

```
>>> cache.keys('*')
['views.decorators.cache.cache_header.api_schema_4.1.7.13748763de6e90ce226f69bcc045cbbb.en.UTC', 'dashboard_rss_628e2b38c2cec0d4aee7cbf72ba189194251c7fc25bd5e1210115aec39a705f4', 'views.decorators.cache.cache_page.api_schema_4.1.7.GET.13748763de6e90ce226f69bcc045cbbb.5c4af095dde9a1645c6d4c7f96a07693.en.UTC']
>>> cache.delete("*api_schema_*")
False
>>> cache.keys('*')
['views.decorators.cache.cache_header.api_schema_4.1.7.13748763de6e90ce226f69bcc045cbbb.en.UTC', 'dashboard_rss_628e2b38c2cec0d4aee7cbf72ba189194251c7fc25bd5e1210115aec39a705f4', 'views.decorators.cache.cache_page.api_schema_4.1.7.GET.13748763de6e90ce226f69bcc045cbbb.5c4af095dde9a1645c6d4c7f96a07693.en.UTC']
```

I don't think wildcard matching is supported. Deleting by the complete key name works:

```
>>> cache.delete("views.decorators.cache.cache_header.api_schema_4.1.7.13748763de6e90ce226f69bcc045cbbb.en.UTC")
True
```

Reply:
Ah, thanks. I was going by the pattern from the script in the thread and presuming that had been verified to work.